common/src/main/java/com/redhat/cloud/notifications/models/NotificationHistory.java

```diff
@@ -187,9 +200,13 @@ public static NotificationHistory getHistoryStub(Endpoint endpoint, Event event,
         history.setEndpointType(endpoint.getType());
         history.setEndpointSubType(endpoint.getSubType());
         history.setEvent(event);
-        history.setInvocationResult(false);
+        history.setStatus(NotificationStatus.FAILED_INTERNAL);
```

Comment:
I think we could/should get rid of that line.

Reply:
I'm wondering if we should have a default to prevent NPE - or maybe just validate that we have some status before saving it.

Reply:
Then let's make the `status` field `@NotNull` maybe.